wrangler-service/src/main/java/io/cdap/wrangler/PropertyIds.java

```diff
@@ -20,6 +20,8 @@
  * This class {@link PropertyIds} is a collection of static strings.
  */
 public final class PropertyIds {
+  // Id of the workspace
```

Comment:
None of the other services use this now, so not sure if this is needed.

Reply:
Ah.